.editorconfig

```diff
@@ -0,0 +1,14 @@
+root = true
+
+[*]
+end_of_line = lf
```

Comment:
Is that for Windows users? :)

Reply:
Yes, but mainly to keeps it consistent regardless of OS used by the developer.